docs/PyOp.md

```diff
@@ -1,6 +1,6 @@
 # Python Operator 
 
-**Deprecation Note: This feature is deprecated and no longer supported.**
+**Deprecation Note: This feature is deprecated and no longer supported, please refer to [onnxruntime_customops](https://github.com/microsoft/ort-customops) project for this function.**
```

Comment:
Can you remind me if ort-customops allows someone to write a custom op in Python and then register it with ORT? How does this work? Is there documentation?

Reply:
Yes, it already implemented. check this tutorial for details
https://github.com/microsoft/ort-customops/blob/main/tutorials/tf2onnx_custom_ops_tutorial.ipynb
in section "Implementing the op in python"
@pranavsharma